adapters/play/src/main/scala/caliban/PlayJson.scala

```diff
@@ -0,0 +1,67 @@
+package caliban
+
+import caliban.interop.play.json.parsingException
+import play.api.libs.json.{ JsObject, JsString, JsValue, Json }
+import play.api.http.Writeable
+import play.api.libs.json.Writes
+
+import scala.util.Try
+
+trait PlayJson {
+
+  implicit def writableGraphQLResponse[E](implicit wr: Writes[GraphQLResponse[E]]): Writeable[GraphQLResponse[E]] =
+    Writeable.writeableOf_JsValue.map(wr.writes)
+
+  private def parseJson(s: String): Try[JsValue] =
+    Try(Json.parse(s))
+
+  def parseHttpRequest(
```

Comment:
There should already be a reads for this.

Reply:
I was using this function to get the GraphqlRequest from query params, probably the name wasn't intuitive. I've replaced it with https://github.com/ghostdogpr/caliban/pull/382/files#diff-e5658f60c6f05951486935dea5234209R29